tools/menuconfig.py

```diff
@@ -249,8 +249,8 @@ def menuconfig(RTT_ROOT):
 
     touch_env()
     env_dir = get_env_dir()
-
-    os.environ['PKGS_ROOT'] = os.path.join(env_dir, 'packages')
+    if isinstance(env_dir, str):
```

Comment:
感觉这里直接判断None就可以了：https://github.com/RT-Thread/rt-thread/blob/eafcdd0bc2e9f1282d100c5b182698a4c9a00b8c/tools/menuconfig.py#L128

```suggestion
    if env_dir != None:
```